pkg/api/apis/installplan/v1alpha1/types.go

```diff
@@ -35,6 +35,7 @@ const (
 type InstallPlanSpec struct {
 	ClusterServiceVersionNames []string `json:"clusterServiceVersionNames"`
 	Approval                   Approval `json:"approval"`
+	Approved                   bool     `json:"approved"`
```

Comment:
This needs to be propagated to the CRD schema as well

Reply:
Which is the correct file with the `InstallPlan-v1` CRD? And how do I express that `approved` is only required if `approval: Manual`?

Reply:
The json-schema way is:

```json
  "dependencies": {
    "foo": ["bar"]
  }
```
(if foo is present, bar is required)

but IIRC the subset used for CRD validation doesn't allow that. It's worth checking, at least.

The canonical `InstallPlan` lives in `deploy/chart/templates`